### DISCUSSION TITLE
Can we implement the following saga pattern structure in workflow?

### DISCUSSION THREAD

**Discussion by anirbanmahapatra** [Netflix/conductor#3365] [Q&A]
I want to implement saga pattern to our current workflow. As per the given image when the syntax_validation_check DECISION fails I want afterwards the revert_syntax_validation task to goto/point-to the revert_instruction_validation and then terminate. Is it possible to achieve this if then how? 
![sagatoknow](https://user-images.githubusercontent.com/81639060/204753042-01236cd4-7e89-4a5f-b7b9-e48313f595ac.PNG)

**Comment by v1r3n** [comment]
Hi @anirbanmahapatra this is possible.  You can put the flow inside a do_while loop to achieve this, where the loop terminates only upon the successful completion of syntax_check_validation, otherwise it repeats the loop.
